Tests/IntegrationTests/Card Tests/DualBrandViewTests.swift

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import XCTest
+@_spi(AdyenInternal) @testable import Adyen
+@testable @_spi(AdyenInternal) import AdyenCard
+
+final class DualBrandViewTests: XCTestCase {
+    var sut: FormCardNumberItemView.DualBrandView!
+    private var imageLoader: MockImageLoader!
+    var brandSelectionCount: Int!
+    
+    override func setUp() {
+        super.setUp()
+        
+        brandSelectionCount = 0
+        imageLoader = MockImageLoader()
+        sut = FormCardNumberItemView.DualBrandView(
+            style: brandImageStyle,
+            imageLoader: imageLoader
+        ) { _ in
+            self.brandSelectionCount += 1
+        }
+    }
+    
+    override func tearDown() {
+        sut = nil
+        imageLoader = nil
+        brandSelectionCount = nil
+        super.tearDown()
+    }
+    
+    func testUpdateCurrentLogos_WhenResettingLoadedImages_ShouldResetToPlaceholder() {
```

Comment:
Let me know if this guy looks too big and it is not easy to follow. If so, I'll break it to 3-4 separate tests.